returnn/log.py

```diff
@@ -192,6 +192,15 @@ def init_by_config(self, config):
                 fn_ext = ".horovod-%i-%i%s" % (hvd.rank(), hvd.size(), fn_ext)
                 new_logs.append(fn_prefix + fn_ext)
             logs = new_logs
+
+        if config.is_true("use_DDP"):
+            new_logs = []
+            for fn in logs:
+                fn_prefix, fn_ext = os.path.splitext(fn)
+                fn_ext = ".nccl-%i-%i%s" % (int(os.environ["LOCAL_RANK"]), int(os.environ["WORLD_SIZE"]), fn_ext)
```

Comment:
Why "nccl"? Call it "torch-distrib" or so. Also, do use `os.environ`.

Reply:
I have to use `os.environ` to get the local rank, the `get_rank()` function from torch.distributed return rank and I couldn't find other function from torch.distributed to get the local rank